en/docs/reference/config-catalog.md

```diff
@@ -949,6 +966,23 @@ claims_extractor_impl = "org.wso2.carbon.apimgt.impl.token.DefaultClaimsRetrieve
                             </div>
                         </div>
                     </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section>
+        <div class="mb-config-options">
+            <div class="superfences-tabs">
+
+                <input name="7" type="checkbox" id="_tab_32">
+                <label class="tab-selector" for="_tab_32"><i class="icon fa fa-code"></i></label>
+                <div class="superfences-content">
+                    <div class="mb-config-example">
+                        <pre><code class="toml">[apim.jwt.issuer.jwks]
```

Comment:
Aren't double brackets `[[ ]]` are incorrect here?
<img width="796" alt="Screenshot 2020-10-22 at 10 50 48" src="https://user-images.githubusercontent.com/5195851/96828432-4b1e8880-1455-11eb-927a-0a67ece5c6e7.png">

Reply:
Double brackets are added because we can have multiple such configs